libs/user-sdk-event/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>it.pagopa.selfcare</groupId>
+        <artifactId>user-libs</artifactId>
+        <version>0.0.1</version>
+    </parent>
```

Comment:
it might be better to avoid relation between apps version with sdk version...sdk version should increment independently, look at https://github.com/pagopa/selfcare-onboarding/blob/main/libs/onboarding-sdk-pom/pom.xml where parent is not present